### PR TITLE
Selenium documentation

### PR DESCRIPTION
**Pull request by anais-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/selenium-test/docs/

File: _docs/learn-by-example/general/selenium-test.md [hunk 1]
```diff
@@ -8,68 +8,168 @@ redirect_from:
...
 toc: true
 ---
-Using this repository, we'll help you get up to speed with basic functionality such as: compiling, testing and building Docker images.
 
-This project uses `JavaScript, Selenium, Protractor` to build an application which will eventually become a distributable Docker image.
- 
-## Looking around
-In the root of this repository you'll find a file named codefresh.yml, this is our build descriptor and it describes the different steps that comprise our process. Let's quickly review the contents of this file:
+[Selenium](https://www.selenium.dev/) is a free, open-source testing framework that is used to automate browser testing. It is encompassed of three different tools, referred to as Selenium Testing.
+* Selenium WebDriver
```
Comment (kostis-codefresh) [inline _docs/learn-by-example/general/selenium-test.md:13]: Give some more details on each component
webdriver = how to control a browser
IDE = graphical tools for devs
grid = cloud platform for running tests

or something similar....

File: _docs/learn-by-example/general/selenium-test.md [hunk 2]
```diff
@@ -8,68 +8,168 @@ redirect_from:
...
-  `codefresh.yml`
+The languages supported by Selenium include Java, Perl, Ruby C#, Python, and JavaScript.
+
+In this example, we are going to be looking at the Selenium WebDriver in combination with [Protractor](https://www.protractortest.org/#/). Protractor is another testing framework specifically for Angular. It simulates the interaction that a user would have within your browsers.
+
+In the end, we will have a container image that executes the tests specified through Protractor on our website.
+
+{% include image.html 
+  lightbox="true" 
+  file="/images/learn-by-example/selenium/selenium-pipeline.png"
```
Comment (kostis-codefresh) [inline _docs/learn-by-example/general/selenium-test.md:25]: pictures do not show up correctly in the preview environment. Can you check the URLs again?

File: _docs/learn-by-example/general/selenium-test.md [hunk 3]
```diff
@@ -8,68 +8,168 @@ redirect_from:
...
 
-{{site.data.callout.callout_info}}
-##### Example
+## What to read next
 
-Just head over to the example [__repository__](https://github.com/codefreshdemo/cf-example-selenium-test){:target="_blank"} in GitHub and follow the instructions there. 
-{{site.data.callout.end}}
+* [Codefresh YAML]({{site.baseurl}}/docs/codefresh-yaml/what-is-the-codefresh-yaml/)
+* [Pipeline steps]({{site.baseurl}}/docs/codefresh-yaml/steps/)
+* [Creating pipelines]({{site.baseurl}}/docs/configure-ci-cd-pipeline/pipelines/)
```
Comment (kostis-codefresh) [inline _docs/learn-by-example/general/selenium-test.md:174]: Link instead to these pages: 
unit tests
integration tests
service container
test reports

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/selenium-test/docs/

File: _docs/learn-by-example/general/selenium-test.md [hunk 4]
```diff
@@ -8,68 +8,168 @@ redirect_from:
...
+  file="/images/learn-by-example/selenium/output.png" 
+  url="/images/learn-by-example/selenium/output.png" 
+  alt="Test Output" 
+  caption="Test Output" 
+  max-width="80%"
+%}
+
+## Running Tests within Codefresh
+
+The [Dockerfile](https://github.com/codefresh-contrib/selenium-example/blob/master/Dockerfile) within our repository creates a container images that sets up the necessary dependencies needed to run our tests. Once we have build and run our our container image, the tests will be executed within. This way, the tests on your application can be run within the container image in your Codefresh Pipeline.
```
Comment (kostis-codefresh) [inline _docs/learn-by-example/general/selenium-test.md:114]: Change this to "Once we have built and run our container image"

**Pull request closed without merging**